api/models/user_group.py

```diff
@@ -8,6 +8,9 @@ class UserGroupQuery(BaseQuery):
     def get_by_uuid(self, uuid):
         return self.filter(UserGroup.group_uuid == uuid).first()
 
+    def get_by_user_phone(self, phone):
+        return self.filter(UserGroup.user_phone == phone).all()
+
```

Comment:
Are we checking last 10 digits?

Reply:
yes the phone number is being sanitize at the beginning